packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
Can we use Repository without a Model? I don't think so, but feel free to prove me wrong :)

IMO, the section "Basic use" should show a minimal code snippet that actually works on its own. We don't necessarily have to explain the underlying concepts right in "Basic use", we can tell people to read more below.

As I mentioned in https://github.com/strongloop/loopback-next/pull/464/files#r130343398, according to our README guidelines, the Overview section should be short. I feel the current version make the Overview section way too long, which pushes installation & basic use out of sigh.

I would personally place the explanation of different concepts (Repository, etc.) in a new section between "Basic Use" and "API Documentation".

Thoughts?